app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
Missing function annotation for the `friends` parameter. It would be a good practice to annotate the type of `friends` parameter for better readability and understanding of the code. For example, `friends: List[Dict[str, Union[str, bool]]]` if each friend is represented as a dictionary.